github-review.el

```diff
@@ -142,6 +143,18 @@ CALLBACK is called with the result"
 CALLBACK is called with the result"
   (github-review-get-pr pr-alist t callback))
 
+(defun github-review-get-pr-deferred (pr-alist needs-diff)
+  "Get a pull request or its diff.
+PR-ALIST is an alist representing a PR,
+NEEDS-DIFF t to return a diff nil to return the pr object
+return a deferred object"
+  (let ((d (deferred:new #'identity)))
+    (if needs-diff
+        (github-review-get-pr-diff pr-alist (apply-partially (lambda (d v &rest _)  (deferred:callback-post d v)) d))
+        (github-review-get-pr-object pr-alist (apply-partially (lambda (d v &rest _)  (deferred:callback-post d v)) d)))
```

Comment:
Oh, interesting, I've always indented the else block differently; haven't seen this style before.

Reply:
So how would you do it ideally in that case? Is there an agreed upon style for emacs-lisp?

Reply:
Or, nothing in particular really, just that when I highlight that region and press `TAB` the indentation changes :-)